metecho/api/models.py

```diff
@@ -332,6 +318,7 @@ class Project(
     # }
     org_config_names = models.JSONField(default=list, blank=True)
     currently_fetching_org_config_names = models.BooleanField(default=False)
+    currently_fetching_github_users = models.BooleanField(default=False)
```

Comment:
We were already disabling the btn on the front end, but it wouldn't persist -- so this is probably better 👍